### PR TITLE
docs(level): expands design guidelines for level layout.

### LINKED ISSUE #3586: Enhance design guidelines for our Layouts pages - Level
Add guidelines to Level Layouts page
Include picture example of Level being used

### PR DESCRIPTION
**Pull request by edonehoo**
Closes #3586

### CODE REVIEW

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md [hunk 1]
```diff
@@ -2,4 +2,23 @@
...
+The **level** layout is designed to distribute space evenly between sections of content and center them horizontally. 
+
+## Elements
+
+By default, level items are placed on the page horizontally and wrap. The level layout is not responsive, but items will wrap as the viewport is resized.
+
+[add image]
+
+1. **Level:** The level container, which holds all level items.
+1. **Level item:** An individual section of content to be placed in the level.
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md:14]:
```suggestion
1. **Level item:** An individual section of content to be placed in the level container. 
```

**Comment by edonehoo** [comment]
@lboehling @mmenestr

**Comment by mmenestr** [comment]
@edonehoo This looks good but it seems to be missing the images still, since I still see "[add image]" in the text haha

**Comment by edonehoo** [comment]
@mmenestr ah yes I've asked many people and no one has been able to supply an image for this :( we could publish without one and then do a followup to keep pursuing one?

**Comment by mmenestr** [comment]
@edonehoo sure that's fine by me!

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md [hunk 2]
```diff
@@ -2,4 +2,23 @@
...
 section: layouts
 ---
-Use a **Level** layout to evenly distribute content horizontally within a container.
+The **level** layout is designed to distribute space evenly between sections of content and center them horizontally. 
+
+## Elements
+
+By default, level items are placed on the page horizontally and wrap. The level layout is not responsive, but items will wrap as the viewport is resized.
+
+[add image]
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md:11]:
```suggestion
```

File: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md [hunk 3]
```diff
@@ -2,4 +2,23 @@
...
+
+## Usage
+
+The level layout can be used to ensure that sections of content are neatly and evenly displayed across the width of a UI.
+
+## Variations
+
+To adjust spacing between items, you can apply gutters.
+
+[add image]
```
Comment (edonehoo) [inline packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/level.md:24]:
```suggestion
```

**Comment by edonehoo** [comment]
@mmenestr cool! How does my change look? I can remove the whole element section if it's weird without an image - just thought the page would be super bare if I did

**Review by mmenestr — APPROVED** [review]
LGTM, merging!

**Pull request merged**